### DISCUSSION TITLE
Priority/Precedence configuration for negative states.

### DISCUSSION THREAD

**Discussion by Windblade-GR01** [ikemen-engine/Ikemen-GO#836] [Feature Request]
SibilantScream recommended that now that we have unlimited negative states it ill be useful to be able to set Priority/Precedence for them.
#125

Priority/Precedence = The order that they are executed.
A implementation could go like this:
```
[Statedef -2]
Priority = -52
```
With the default begin 0.

**Comment by potsmugen** [comment]
The way they run in ascending order is probably more elegant than priority. Can always move code from one to the other for it to run later.
I think State +1 solved that user's original issue, too.
